config/elvis.config

```diff
@@ -26,6 +26,11 @@
                     #{regex => "^([a-z][a-z0-9]*_?)*(_SUITE)?$",
                       ignore => []}
                   },
+                  {
+                    elvis_style,
+                    function_naming_convention,
+                    #{regex => "^([a-z]*_?)*(_SUITE)?$"}
```

Comment:
Sure. pushing the change now, and will do that PR for the guidelines after that. Thanks.